benches/benches-outputs/Cargo.toml

```diff
@@ -12,3 +12,6 @@ fuel-core-types = { path = "../../crates/types", default-features = false, featu
 ] }
 
 [workspace]
+
+[patch.crates-io]
```

Comment:
And not this either

Reply:
It is here since `fuel-vm` is not released yet=D I created PR in advance. It is ready for review and only waits for a release.